src/Bicep.Core.Samples/Files/InvalidParameters_LF/main.diagnostics.bicep

```diff
@@ -412,6 +412,33 @@ param commaTwo string {
     default: 'abc'
 }
 
+@secure
+//@[1:7) [BCP063 (Error)] The name "secure" is not a parameter, variable, resource or module. |secure|
+@
+//@[1:1) [BCP123 (Error)] Expected a namespace or decorator name at this location. ||
+param incompleteDecorators string
+
+@concat(1, 2)
```

Comment:
>concat [](start = 1, length = 6)

Should also include sys.concat() case too.

Reply:
Good point. Will add it.

Reply:
Done.